volatility/framework/plugins/windows/strings.py

```diff
@@ -5,6 +5,7 @@
 import logging
 import re
 from typing import Dict, Generator, List, Set, Tuple
+from os.path import getsize
```

Comment:
Typically (save for the typing module which is used literally everywhere), we tend to do `from os import path` and then do `path.getsize`.  This helps ensure that no one does `from volatility.plugins.strings import getsize` because it's easy and then it breaks later on.

Reply:
Great tip! thanks 👍 
I updated the code so you are more than welcomed to merge 🥇 !